onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -208,6 +208,14 @@ def symbolic_value(self) -> torch.Value:
         """The symbolic Value in torch.Graph."""
         return self._torch_value
 
+    def value_info(self) -> Optional[onnx.ValueInfoProto]:
+        try:
+            dtype = self.onnx_dtype.value
+        except torch.onnx.errors.OnnxExporterError:
```

Comment:
Curious: why do we get OnnxExporterError?

Reply:
Got it from below in torch/onnx/_type_utils.py
```
        if isinstance(value.type(), torch.ListType):
            try:
                return cls.from_dtype(value.type().getElementType().dtype())
            except RuntimeError:
                return cls._from_name(str(value.type().getElementType()))
```

Reply:
Any chance we can take the logic out and remove dependence on type utils?

Reply:
I'm not following, would you prefer a general exception class? In general, we want this method to return None if dtype is not retrievable or undefined.

Reply:
I am just thinking we should move away from using `_type_utils.JitScalarType` from torch.onnx. But this can be done separately